examples/time_series_forecasting.ipynb

```diff
@@ -0,0 +1,465 @@
+{
```

Comment:
<p>Would it be too much to ask to visualise the time series? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/giotto-ai/giotto-tda/pull/475/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Even if they are kept as they are, i.e. ridiculously basic? Might it not affect how "serious" the notebook looks if we keep plotting straights lines?

Reply:
<p>ah yes you are right. let's not bother plotting them for now since it's not really necessary to understand what's going on </p>